Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -472,8 +476,12 @@ void switchStructure(Object treeNodeData) throws InvalidMetadataValueException,
         }
     }
 
-    void switchMediaUnit() throws InvalidMetadataValueException, NoSuchMetadataFieldException {
-        metadataPanel.preservePhysical();
+    void switchMediaUnit() throws NoSuchMetadataFieldException {
+        try {
+            metadataPanel.preservePhysical();
+        } catch (InvalidMetadataValueException e) {
+            logger.info(e.getLocalizedMessage());
```

Comment:
Stack trace is nice to have (but not necessary):

`logger.info(e.getLocalizedMessage(), e);`